docs/firefighting/index.md

```diff
@@ -542,8 +542,12 @@ WHERE nspname NOT IN ('pg_catalog', 'information_schema')
 GROUP BY main_table
 ORDER BY n_tup_ins DESC;
 ```
-
-
+### Disk Full on Data partition
+In Case Postgresql fails with  `No space left on device` Error message and in order to free up space you need to restart the Postgresql then Please take the following steps
```

Comment:
```suggestion
In Case PostgreSQL fails with  `No space left on device` error message and in order to free up space needed to restart the PostgreSQL then take the following steps
```